packages/libs/error-stack/src/fmt/color.rs

```diff
@@ -0,0 +1,161 @@
+use crate::{
+    fmt::r#override::{AtomicOverride, AtomicPreference},
+    Report,
+};
+
+/// The available modes of color support
+///
+/// Can be accessed through [`crate::fmt::HookContext::color_mode`], and set via
+/// [`Report::set_color_mode`]
+#[derive(Debug, Copy, Clone, Eq, PartialEq)]
```

Comment:
We should derive `Hash` as well

Reply:
fixed in [a727237](https://github.com/hashintel/hash/pull/1800/commits/a727237a55d6a1fdcad4abc805e546800833dd14)